src/components/Welcome/Welcome.js

```diff
@@ -1,56 +1,60 @@
 import React from 'react';
 import RecipeCard from '../RecipeCard/RecipeCard.js';
 import './Welcome.css';
-import { getRandomRecipe } from './../../APICalls'
+import { getRandomRecipe, getLatestRecipes } from './../../APICalls'
+import LoginButton from '../LoginButton/LoginButton';
+
 
 const Welcome = () => {
   const  userData = {}
```

Comment:
By looking at this line of `userData`, I'm not sure what its purpose is. Will it be used as state once the user is logged in?